lms/djangoapps/dashboard/sysadmin_get_course_users.py

```diff
@@ -269,11 +271,11 @@ def get_report_data_for_course_users(courses, course_id):
                 status_list.append(status)
         # some default users could be without last_login, so:
         try:
-            last_login = u.last_login.strftime('%Y-%m-%d %H:%M')
+            last_login = localtime(u.last_login).strftime('%Y-%m-%d %H:%M')
         except Exception:
```

Comment:
@dyudyunov Is it possible to narrow down this exception to something more specific?